anybox/recipe/odoo/base.py

```diff
@@ -1663,17 +1665,17 @@ def insert_odoo_git_addons(self, base_addons):
         except ValueError:
             addons_paths.insert(insert_at, odoo_git_addons)
 
-    def cleanup_openerp_dir(self):
+    def cleanup_odoo_dir(self):
         """Revert local modifications that have been made during installation.
 
         These can be, e.g., forbidden by the freeze process."""
 
-        # from here we can't guess whether it's 'openerp' or 'odoo'.
+        # from here we can't guess whether it's 'odoo' or 'odoo'.
         # Nothing guarantees that this method is called after develop().
         # It is in practice now, but one day, the extraction as a separate
         # script of freeze/extract will become a reality.
-        for proj_name in ('openerp', 'odoo'):
-            egg_info_dir = join(self.openerp_dir, proj_name + '.egg-info')
+        for proj_name in ('odoo', 'odoo'):
```

Comment:
Remaining glitch from the global search and replace. You will want to keep a check for 'openerp' here.